doc/howtos/software-update-server.rst

```diff
@@ -0,0 +1,281 @@
+   - OSTRO_IMAGE_INSTALL_REFERENCE
+   - IMAGE_INSTALL
+
+   The recommended way is to modify a copy of the original recipe
+   ``ostro-image.bb`` and refer to ``ostro-image.bbclass`` for
+   the values to use.
+
+
+Step 1: Review and optionally modify the list of bundles
+--------------------------------------------------------
+Ostro already defines some bundles, and users can define additional ones.
+
+Example::
+
+  SWUPD_BUNDLES += "sudo_feature"
+  BUNDLE_CONTENTS[sudo_feature] = "sudo"
+
+This example is an alternative to add the component that was shown in
+Step 0.  Instead of extending the content of os-core, put the extra
+component in its own bundle. The main difference is that this
+approach will not install the component by default in any image.
+
+
+Step 2: Review and optionally modify the content of the images created
+----------------------------------------------------------------------
+It is also possible to choose which bundles will be pre-installed on the
+customized image, by:
+
+#. adding the bundle to the list of images generated for bundle purposes
+
+   Example::
+
+     SWUPD_IMAGES += "\
+       pre_installed_content \
+     "
+
+#. adding a varflag to SWUPD_IMAGES.
+
+   Example::
+
+     SWUPD_IMAGES[pre_installed_content]="\
+       sudo_feature \
+       add_here_other_features_if_needed \
+     "
+
+   ``SWUPD_IMAGES`` is the only token which is specific to the Ostro OS
+   syntax. The other identifiers are under the developer's control.
+
+
+This will add to the image content the ``sudo_feature`` feature from Step 1.
+
+The main difference, compared to the approach taken in Step 0 is that
+this approach allows for removing the content of the bundle without having to
+create an update (and thus a new release).
+
+.. note::
+   Each bundle defined generates a non-negligible amount of load, when
+   building images and SW Update repositories.
+
+   It is therefore recommended to keep enabled only those bundles that
+   are effectively useful for the specific use-case targeted.
+
+   In case one wants to modify the set of predefined bundles that come
+   from the defaults of Ostro OS, it is possible to override the variable
+   ``SWUPD_BUNDLES``.
+
+   Doing so will discard anything that was part of the defaults, therefore
+   if any of the pre-defined bundles is still needed, it must be listed
+   again, explicitly.
+
+   Example::
+
+   ``SWUPD_BUNDLES = "sudo_feature"``
+
+
+
+Step 3: Build the images and the SW Update repositories
+-------------------------------------------------------
+
+The typical invocation for generating the basic Ostro OS image and related
+SW Update stream is::
+
+ $ bitbake ostro-image-swupd
+
+However, if one wants to have pre-installed bundles, then the invocation
+must refer to the specific variant.
+
+  Example (Continuing from Step2)::
+
+    $ bitbake ostro-image-swupd-pre_installed_content
+
+Assuming that the chosen architecture was ``intel-corei7-64``, the yield
```

Comment:
Initially I used output, but yield seemed more fitting.
yield: to produce (something) as a result of time, effort, or work.

Given the time/resource intensive process required to generate the images, it seemed fitting :-)

Of course I might be wrong, as I'm not a native English speaker.